src/utils/api_helpers.py

```diff
@@ -27,6 +33,25 @@ def add_object_to_database(obj: Base) -> dict | str:
             logging.error(e)
             return e.orig.args[0]
 
+def is_jwt_valid(jwt_token: str, secret_key: str) -> bool:
+    if not jwt_token:
+        return False
+
+    is_valid = False
+    decoded_jwt=jwt.decode(jwt_token, secret_key, algorithms=["HS256"], verify=True)
```

Comment:
Need a try-catch here to cover the case where it doesn't decode successfully and throws an error. We want the function to still return false in that case, not throw